src/fetch.rs

```diff
@@ -287,27 +291,27 @@ fn get_name_from_manifest(manifest: &Manifest) -> Result<String> {
         .ok_or_else(|| ErrorKind::ParseCargoToml.into())
 }
 
-fn get_default_timeout() -> Duration {
-    Duration::from_secs(10)
-}
-
-fn get_with_timeout(url: &str, timeout: Duration) -> reqwest::Result<reqwest::Response> {
-    let client = reqwest::ClientBuilder::new()
-        .timeout(timeout)
-        .proxy(reqwest::Proxy::custom(|url| {
-            env_proxy::for_url(url).to_url()
-        }))
-        .build()?;
-
-    client
+fn get_with_default_timeout(url: &str) -> reqwest::Result<reqwest::Response> {
+    lazy_static! {
+        static ref CLIENT: reqwest::Client = build_client();
```

Comment:
this will build a client with proxy set to `env_proxy::for_url(url).to_url()`, where `url` is the first `url` called by `get_with_default_timeout`. I don't think that's the expected behavior.

Reply:
You are right, that's my bad, and I don't know how to fix this. Maybe I should close this pull request.